Backend/admin/src/components/login/login.js

```diff
@@ -3,60 +3,74 @@ import { Button, Form, Input } from 'antd';
 import { useHistory } from 'react-router';
 import { login } from '../../db/auth';
 
+import './login.css';
+
 const layout = {
-  labelCol: { span: 8 },
+  labelCol: { span: 6 },
   wrapperCol: { span: 16 },
 };
 const tailLayout = {
-  wrapperCol: { offset: 8, span: 16 },
+  wrapperCol: { offset: 6, span: 16 },
 };
 
 const Login = () => {
   const history = useHistory();
 
+  const [form] = Form.useForm();
```

Comment:
@RafaelClaudiu  @razvanip I am getting a lot of warnings since we are using StrictMode. I don't know if it is ok.

Reply:
Warnings of ?

Reply:
If you are referring to this warning "The old API will be supported in all 16.x releases," I don't think is from our project.

And the warnings from eslint some of them and the majority are about prop types and console.log. 

So it's oki for now